.pylintrc

```diff
@@ -62,6 +62,7 @@ ignore-imports=yes
 #    too-many-statements
 # and fix those issues.
 
+disable=no-self-use,broad-except,too-many-statements,too-many-locals,too-many-branches,dangerous-default-value,not-context-manager,duplicate-code
```

Comment:
@prasanna08 did you see this comment? It would be helpful if you could reply "Done" to the comments you've addressed, following the instructions in step 5 [here](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change)? This helps keep track of which comments have been addressed and which haven't.